inc/options/cdn.php

```diff
@@ -377,8 +377,11 @@
 
 			<tr>
 				<th colspan="2">
-					<?php $this->checkbox( 'cdn.flush_manually' ); ?>
 					<?php
+					$this->checkbox(
+						'cdn.flush_manually',
+						Cdn_Util::get_flush_manually_default_override( $this->_config->get_string( 'cdn.engine' ) )
```

Comment:
This argument disables the checkbox.  A user should be able to toggle the setting regardless of the `cdn.engine` selected.
See https://github.com/W3EDGE/w3-total-cache/blob/5a84ed9f83d7a4fe18c4d59453012b1f444eb4d6/Base_Page_Settings.php#L88

Reply:
Reverted and proper application added to "checkbox" function to override the default boolean for cdn.flush_manually